src/ios-deploy/ios-deploy.m

```diff
@@ -1714,7 +1714,7 @@ void list_bundle_id(AMDeviceRef device)
                          @"CFBundleVersion",
                          @"CFBundleShortVersionString", nil];
     if (key) {
-        NSArray * ns_keys = [[NSString stringWithUTF8String:key] componentsSeparatedByString:@"&"];
+        NSArray * ns_keys = [[NSString stringWithUTF8String:key] componentsSeparatedByString:@","];
```

Comment:
This will break anyone relying on the current behavior. I agree that "," is more appropriate, but I'm against merging this as-is. What might be better is to support multiple of the same argument to build an array.

e.g.
```ios-deploy -B -j --key=UIFileSharingEnabled&CFBundlePackageType```
 would become 
```ios-deploy -B -j --key=UIFileSharingEnabled --key=CFBundlePackageType```
that wouldn't break people.

Reply:
Thank you for the advice , please take look at the latest commit .